plugins/ezpz-rust-ti/python/ezpz_rust_ti/__init__.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+
+def register_plugin() -> dict[str, Any]:
```

Comment:
we should use something with narrower typing than dict[str, Any] for users to register a plugin. most likly a pydantic model so that we can also run validations at the same time via the model